### PR TITLE
CXGFF-357: new stringify endpoint

### PR DESCRIPTION
**Pull request by mvollmar**
## Summary

Add new stringify endpoint that works without a viewId 

## Checklist

-   [X] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] My changes generate no new warnings (ESLint)

### CODE REVIEW

File: src/service/EntityService.ts [hunk 1]
```diff
@@ -87,15 +87,24 @@ export class EntityService extends APIClient {
...
         return this.invokeApi(`/views/${viewId}/schemas/${schemaId}/entities/${entityId}/stringify`);
     }
 
+    /**
+     * Get a stringified entity by type instead of viewId
+     * @param schemaId
+     * @param entityId
+     */
+    async stringifyEntityByType(schemaId: string, entityId: string): Promise<AxiosResponse<string | undefined>> {
+        return this.invokeApi(`/views/schemas/${schemaId}/entities/${entityId}/stringify`);
```
Comment (ffjanhoeck) [inline src/service/EntityService.ts:105]:
```suggestion
     * @param schemaName
     * @param entityId
     */
    async stringifyEntityByType(schemaName: string, entityId: string): Promise<AxiosResponse<string | undefined>> {
        return this.invokeApi(`/views/schemas/${schemaName}/entities/${entityId}/stringify`);
```

File: src/service/EntityService.ts [hunk 2]
```diff
@@ -87,15 +87,24 @@ export class EntityService extends APIClient {
...
 
     /**
-     * TODO: Please comment this method
+     * Get a stringified entity by viewId
      * @param schemaId
      * @param entityId
      * @param viewId
      */
-    async stringifyEntity(schemaId: string, entityId: string, viewId: string = 'EntityRelationView'): Promise<AxiosResponse<any>> {
+    async stringifyEntity(schemaId: string, entityId: string, viewId: string = 'EntityRelationView'): Promise<AxiosResponse<string | undefined>> {
```
Comment (ffjanhoeck) [inline src/service/EntityService.ts:95]: Mape add @deprecated

File: src/service/EntityService.ts (hunk 1 above)
Comment (mvollmar) [reply src/service/EntityService.ts]: Applied it 👍

File: src/service/EntityService.ts (hunk 2 above)
Comment (mvollmar) [reply src/service/EntityService.ts:96]: Makes sense, added it 👍

**Review by ffjanhoeck — APPROVED** [review]

**Pull request merged**